### PR TITLE
Fixed Broken Link to 'What is Infrastructure as Code'

### LINKED ISSUE #24380: Broken link
The first link to Infrastructure as Code is broken (https://docs.microsoft.com/en-us/azure/devops/learn/what-is-infrastructure-as-code).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 09be1939-e2a5-241e-4114-8747b35f3d21
* Version Independent ID: 7d77200d-9764-cabc-d5c1-c4d2ba3c2fd1
* Content: [Infrastructure as code](https://docs.microsoft.com/en-us/dotnet/architecture/cloud-native/infrastructure-as-code)
* Content Source: [docs/architecture/cloud-native/infrastructure-as-code.md](https://github.com/dotnet/docs/blob/main/docs/architecture/cloud-native/infrastructure-as-code.md)
* Product: **dotnet-architecture**
* Technology: **cloud-native**
* GitHub Login: @nishanil
* Microsoft Alias: **nanil**

### PR DESCRIPTION
**Pull request by scottstout**
## Summary

Describe your changes here.
Note that this fix is a little weird since the source mentions the author (Sam Guckenheimer) by name, but the destination is just documentation that doesn't have an author attributed. The quoted text is in the destination location so I'm sure this is the correct destination.

Fixes #24380

### CODE REVIEW

File: docs/architecture/cloud-native/infrastructure-as-code.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Figure 10-17 shows a YAML snippet that lists the version of Azure CLI and the de
...
+In the article, [What is Infrastructure as Code](https://docs.microsoft.com/en-us/devops/deliver/what-is-infrastructure-as-code), Author Sam Guckenheimer describes how, "Teams who implement IaC can deliver stable environments rapidly and at scale. Teams avoid manual configuration of environments and enforce consistency by representing the desired state of their environments via code. Infrastructure deployments with IaC are repeatable and prevent runtime issues caused by configuration drift or missing dependencies. DevOps teams can work together with a unified set of practices and tools to deliver applications and their supporting infrastructure rapidly, reliably, and at scale."
```
Comment (Youssef1313) [inline docs/architecture/cloud-native/infrastructure-as-code.md:122]:
```suggestion
In the article, [What is Infrastructure as Code](/devops/deliver/what-is-infrastructure-as-code), Author Sam Guckenheimer describes how, "Teams who implement IaC can deliver stable environments rapidly and at scale. Teams avoid manual configuration of environments and enforce consistency by representing the desired state of their environments via code. Infrastructure deployments with IaC are repeatable and prevent runtime issues caused by configuration drift or missing dependencies. DevOps teams can work together with a unified set of practices and tools to deliver applications and their supporting infrastructure rapidly, reliably, and at scale."
```

**Review by Youssef1313 — COMMENTED** [review]
Can you please mark as "Fixes https://github.com/dotnet/docs/issues/24380" in the PR description?

**Comment by scottstout** [comment]
Done

**Comment by gewarren** [comment]
@scottstout Thanks for fixing this. This was already fixed in the file you changed and in a couple other places in https://github.com/dotnet/docs/pull/24650.

**Pull request closed without merging**